autoapi/extension.py

```diff
@@ -270,6 +270,7 @@ def setup(app):
     app.add_config_value("autoapi_python_class_content", "class", "html")
     app.add_config_value("autoapi_generate_api_docs", True, "html")
     app.add_config_value("autoapi_prepare_jinja_env", None, "html")
+    app.add_config_value("autoapi_render_in_single_page", None, "html")
```

Comment:
Just a very interested reader here, as a user I would very much prefer to set-up the "level" I want to see in single page rather than a list of types. As you mentioned it would be very easy to forget the values that are higher in the tree.

Reply:
Sounds good to me, @AWhetter @12rambau. In that case, I wanna propose to rename the option to `autoapi_single_page_level` if it works for you.

This is also related with the renaming Ashley suggested.